internal/store/serverservice.go

```diff
@@ -120,3 +120,8 @@ func (s *Serverservice) BillOfMaterialsBatchUpload(ctx context.Context, boms []s
 func (s *Serverservice) GetBomInfoByAOCMacAddr(ctx context.Context, macAddr string) (*sservice.Bom, *sservice.ServerResponse, error) {
 	return s.client.GetBomInfoByAOCMacAddr(ctx, macAddr)
 }
+
+// GetBomInfoByBMCMacAddr will return the bom info object by the bmc mac address.
+func (s *Serverservice) GetBomInfoByBMCMacAddr(ctx context.Context, macAddr string) (*sservice.Bom, *sservice.ServerResponse, error) {
```

Comment:
I would like us to follow Idiomatic Go function naming practices https://go.dev/doc/effective_go#Getters,
this can be done in a separate PR as well.

Reply:
Thanks for the suggestion and will add that in a separate PR.